docs/connectors/sinks/iceberg.md

```diff
@@ -99,3 +94,12 @@ Connect.
     "value.converter.schemas.enable": "true"
 }
 ```
+- Ensure that `iceberg.tables.default-id-columns` is consistent with the index
+  definition in Feldera.
+- Ensure that the Iceberg table is accessible from Kafka Connect.
+- Ensure that `iceberg.tables.auto-create-enabled` is set to `true` if the
+  table doesn't already exist.
+- Ensure that `iceberg.tables.evolve-schema-enabled` is set to `true` if you
```

Comment:
- Ensure that `iceberg.tables.evolve-schema-enabled` is set to `true` if you
  want to dynamically update the Iceberg table schema to match the SQL view declaration.